docs/src/tutorials/symbmc.md

```diff
@@ -0,0 +1,655 @@
+Conversely, with respect to `{p1,p2}`, there are four symbolic states: one corresponds to states which are both reachable and violate the invariant, one corresponds to states which are reachable, but which do not violate the invariant, one corresponds to states which are not reachable, but violate the invariant and the last one corresponds to states which are neither reachable, nor violate the invariant.
+
+### Symbolic traces
+Having defined symbolic states, what is then the meaning of a symbolic trace?
+In the explicit model, a _trace_ of length `k` is simply a sequence of reachable states 
+`s0,..., sk \in S`, such that `s0 \in R(0) /\ ... /\ sk \in R(k)` and `si -> s{i+1}`. 
+In the symbolic setting, a _symbolic trace_ is a sequence of symbolic states `C0,...,Ck \in SUBSET S`, such that 
+```
+C0 \in S // ®_P0 /\ ... /\ Ck \in S // ®_Pk
+```
+
+and, for each `i = 0,...k`, it is the case that `Ci = { s \in S: Pi(s) = TRUE }`.
+In other words, a symbolic trace is the unique sequence of symbolic states, which correspond to the set of explicit states evaluating to `TRUE` under each of `P0,...,Pk` respectively.
+
+Recall that `P{i+1}(s{i+1})` was defined as `\E si \in S: Pi(si) /\ si -> s{i+1}`.
+While, in the explicit case, we needed to enforce the condition `si -> s{i+1}`, in the symbolic case this is already a part of the predicate definition.
+
+For example, consider:
+```tla
+VARIABLE 
+  \* @type: Int;
+  x
+
+Init == x \in {0,1}
+Next == x' = x + 1
+```
+a trace of length 2 would be one of `0,1,2` or `1,2,3`. A symbolic trace would be the sequence
+```
+0..1, 1..2, 2..3
+```
+
+<!-- ### Symbolic transitions
+Considering the definition of a symbolic state lifts the analysis from `S` to `S // R`, for an equivalence relation `R` derived from a unary predicate `P` over `S`, it should come as little surprise, that symbolic transitions will be defined in terms of `S \X S // R`, for an equivalence relation `R` derived from a binary predicate `P` over `S \X S`.
+
+Let `P` be a binary predicate over `S \X S`. `P` naturally defines a relation `T_P` over `(S \X S) \X (S \X S)` in the following way:
+```
+T_P(<<s1,s2>>, <<t1,t2>>) <=> P(s1,s2) = P(t1,t2)
+```
+Note that this is the exact same definition we used in the previous section, up to pattern-matching the domain of the predicate (`S`, in the previous section could be `A \X B` for some `A,B`). -->
+
+In the case of symbolic states, we were particularly interested in symbolic states with respect to predicates that encoded reachability. 
+<!-- Here, we take a look at scenarios, where `P(s1,s2)` is `s1 -> s2`. For this reason, a _symbolic transition_ is defined to simply be an element of  `S \X S // T_->`.
+Again, in the sense of the predicate-subset equivalence, a symbolic transition is either `->` or its negation.
+ -->
+Unlike the case of general predicates, the most interesting scenario w.r.t. traces is when a transition relation is presented as a disjunction of transitions, i.e. when 
```

Comment:
How is this unlike general predicates? In the case of general predicates are these scenarios not interesting? Or they are just not the "most interesting"? Or?